string first_name
string second_name